system/Images/Handlers/GDHandler.php

```diff
@@ -472,9 +472,9 @@ protected function textOverlay(string $text, array $options = [], bool $isShadow
         // Add the shadow to the source image
         if (! empty($options['fontPath'])) {
             // We have to add fontheight because imagettftext locates the bottom left corner, not top-left corner.
-            imagettftext($src, $options['fontSize'], 0, $xAxis, $yAxis + $options['fontheight'], $color, $options['fontPath'], $text);
+            imagettftext($src, $options['fontSize'], 0, (int) $xAxis, $yAxis + $options['fontheight'], $color, $options['fontPath'], $text);
```

Comment:
Is the yAxis automatically cast due to the '+'?

Reply:
Good catch! Fixed